src/global_mapper_ros/global_mapper_ros.h

```diff
@@ -3,9 +3,11 @@
 
 #include <csignal>
 #include <memory>
+#include <string>
+#include <vector>
 
 #include "ros/ros.h"
-#include <tf2_ros/transform_listener.h>
+#include "tf2_ros/transform_listener.h"
```

Comment:
Hm...I'm not sure I agree with that interpretation. From the first answer:
> For #include "filename" the preprocessor searches in the same directory as the file containing the directive. This method is normally used to include programmer-defined header files.

> For #include <filename> the preprocessor searches in an implementation dependent manner, normally in search directories pre-designated by the compiler/IDE. This method is normally used to include standard library header files.

My takeaway is that quotes tell the compiler to look in the current directory first, while brackets leave the search path up to the compiler. You shouldn't search in the current directory for external headers, so I think brackets should be used in this case. I'm also not clear about what they mean when they say "system" file, as that seems to suggest anything installed externally (e.g. in `/usr/include` or `rrg/dependencies/install`), but that is very different from headers included with the compiler.